### ISSUE TITLE
MathNet.Numerics.fsx missing from nuget package (v 4.4.1)

### ISSUE THREAD

**Issue by sebhofer** [mathnet/mathnet-numerics#567]
I just upgraded MathNet.Numerics to the latest version in one of my F# projects, and now I'm missing "../packages/MathNet.Numerics.FSharp/MathNet.Numerics.fsx" that I always used to load it. The documentation still mentions this as the prefered way to load MathNet. I also tried referencing the dll's directly, but then I receive an error saying:

error FS0074: The type referenced through 'MathNet.Numerics.LinearAlgebra.Vector`1' is defined in an assembly that is not referenced. You must add a reference to assembly 'MathNet.Numerics'.

Am I missing something?

**Comment by cdrnet** [comment]
Yes, it is indeed missing since v4.0, last release where it was included was v3.20.2.

I assume you use this in F# scripts?

**Comment by sebhofer** [comment]
That's what I as trying to do, yes. For the moment I downgraded to v3.20.2 again.

**Comment by sebhofer** [comment]
So I'm guessing I can just use the old version of MathNet.Numerics.fsx for the moment?

**Comment by sebhofer** [comment]
I'm not sure if this is helpful, but I think Deedle does a similar thing. I just dug into the Deedle repo and found the following structure in the .nuspec file:
```
<?xml version="1.0" encoding="utf-8"?>
<package xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
  <metadata xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
    ...
  </metadata>
  <files>
    ...
    <file src="..\bin\Deedle.fsx" target="." />
  </files>
</package>
```
where Deedle.fsx also contains setup code for the library. I'm guessing (but I don't know how to check) that you could just do the same.

**Comment by cdrnet** [comment]
Thanks. The problem is that I no longer have a nuspec file, as it is generated automatically during `dotnet pack` from the project file metadata.
